### ISSUE TITLE
[apostrophe-cli] Warning: Accessing non-existent property 'padLevels' of module exports inside circular dependency

### ISSUE THREAD

**Issue by mqwerty** [apostrophecms/apostrophe#2469]
## To Reproduce

```bash
npm install apostrophe-cli -g
apos create-project test
> Apostrophe  create-project  Grabbing the boilerplate from Github [1/2]
> Cloning into 'apos'...
> Apostrophe  create-project  Setting up your project shortname [2/2]
> (node:13128) Warning: Accessing non-existent property 'padLevels' of module exports inside circular dependency
> (Use `node --trace-warnings ...` to show where the warning was created)
```

## Details

MacOS X 10.15.7 and Windows10

```bash
apos --version
> apostrophe-cli: v2.3.5
> apostrophe v2.112.1 (latest) will be installed with a new project, unless the boilerplate project's dependencies specify otherwise.

npm --version
> 6.14.8

node --version
> v14.13.1
```

**Comment by abea** [comment]
It looks like this is an issue with Node 14.

**Comment by abea** [comment]
@mqwerty This should be resolved in the Apostrophe CLI `2.3.6` release that we published yesterday. Please update to that and let us know if there are still problems on Node 14.
